resources/helm/overlays/istio-control/istio-discovery/templates/meshclusterrole.yaml

```diff
@@ -11,3 +11,18 @@ rules:
   - apiGroups: ["authentication.k8s.io"]
     resources: ["tokenreviews"]
     verbs: ["create"]
+  # TODO: This is temporary. We need to figure out why this is needed and remove these permissions.
+  - apiGroups:
+    - networking.k8s.io
+    resources:
+    - ingressclasses
```

Comment:
Looks like this was added in 1.9 to replace  `kubernetes.io/ingress.class` annotation.
https://istio.io/latest/docs/tasks/traffic-management/ingress/kubernetes-ingress/#specifying-ingressclass

Reply:
jfyi, we should be disabling the ingress controller in istio.

Reply:
I don't know how to disable the ingress controller. Is there an option in the chart to turn it off?

Reply:
This is good for now. https://github.com/maistra/istio-operator/pull/737#commitcomment-52972441